backstack/build.gradle.kts

```diff
@@ -14,6 +14,18 @@ kotlin {
   jvm()
   ios()
   iosSimulatorArm64()
+  js(IR) {
+    moduleName = property("POM_ARTIFACT_ID").toString()
```

Comment:
This is set to the project name by default which appears to already be unique. Why the override?

Reply:
Mostly for consistency if we move dirs around/nest projects